internals/models_test.py

```diff
@@ -339,10 +339,17 @@ def setUp(self):
     self.feature_1.put()
     self.feature_1_id = self.feature_1.key.integer_id()
     self.appr_1 = models.Approval(
-        feature_id=self.feature_1_id, field_id=1, state=2,
-        set_on=datetime.datetime.now(),
+        feature_id=self.feature_1_id, field_id=1,
+        state=models.Approval.REVIEW_REQUESTED,
+        set_on=datetime.datetime.now() - datetime.timedelta(1),
```

Comment:
Not sure what kinds of clocks it uses for testing, but hope `datetime.datetime.now() - datetime.timedelta(1)` doesn't go below 0?

Reply:
I believe that it uses the regular clock.  It seems to work fine.